### ISSUE TITLE
ERR 520 : unable to reply to self

### ISSUE THREAD

**Issue by iamdey** [CaliOpen/Caliopen#707]
https://feedback.caliopen.org/t/erreur-quand-je-veux-me-repondre-a-moi-meme/316

post the following in a discussion w/ myself:

```
body | réponse censé être en échec
discussion_id | b657986c-026d-4a9b-8a43-139e6d61dfd8
identities | 
  0 |
    identifier | dey@alpha.caliopen.org
    type | email
    parent_id | 43f99507-4154-42e2-b0dc-cf4bfacb0cfe
    subject | test shizo 1
```

resp 

```
{
	"errors": [{
		"code": 520,
		"description": "",
		"component": "collection_message",
		"values": 520,
		"message": "vars() argument must have __dict__ attribute",
		"property": "MergePatchError",
		"type": "Unknown Error"
	}]
}
```

**Comment by gdchamal** [comment]
backend apiv1 logs are :

```
INFO:elasticsearch:GET http://192.168.1.10:9200/96227c19-7639-4789-a934-524cf8e48366/indexed_message/cc9549dd-3942-4bd9-9893-9345581804a9 [status:200 request:0.057s]
WARNING:caliopen_main.message.objects.message:draft_param error
WARNING:caliopen_main.message.objects.message:vars() argument must have __dict__ attribute
WARNING:caliopen_api.message.message:vars() argument must have __dict__ attribute

```

**Comment by sapiens-sapide** [comment]
Difficult to reproduce, but may be fixed by #836ba86.

**Comment by gdchamal** [comment]
Sound fixed, no error found since last deploy
